hermeto/core/resolver.py

```diff
@@ -82,8 +82,7 @@ def _resolve_packages(request: Request) -> RequestOutput:
         for pkg in experimental_types:
             if pkg in _dev_package_managers:
                 _supported_package_managers[pkg] = _dev_package_managers[pkg]
-    elif "dev-package-managers" in request.flags:
-        _supported_package_managers |= _dev_package_managers
+
```

Comment:
I think this should have happened in an earlier commit where the resolver module is adjusted.